File: JS/10-18-25/Java/PerfectNumber2.java
public class PerfectNumber2{
public static void main(String [] args){	


	
	for (int number = 1; number <= 1000; number++){

		int sum = 0;

		for( int divisor = 1; divisor < number; divisor++ ){
			if( number % divisor == 0 )
				sum += divisor;
			}

		if( sum == number )
			System.out.println(number);
		}
}}

File: JS/10-18-25/JS/perfect_number2.js
for (let number = 1; number <= 1000; number++){
	let sum = 0;

	for (let divisor = 1;divisor < number; divisor++){
		if (number % divisor == 0)
			sum += divisor;
	}
	if (sum == number)
		console.log(number);
	